wpilibj/src/main/java/edu/wpi/first/wpilibj/Joystick.java

```diff
@@ -8,7 +8,7 @@
 import edu.wpi.first.hal.HAL;
 
 /**
- * Handle input from standard Joysticks connected to the Driver Station.
+ * Handle input from Flight Joysticks connected to the Driver Station.
```

Comment:
As with the game controller discussion, one of the most common flight sticks is the Extreme 3D Pro, but it's not the only one.  Does this warrant the same mapping treatment?

Reply:
IMO, no. Most buttons on the joystick are read using an index-taking method anyway, and the few "named" button/axes can be configured using `setXxxChannel()`. We don't do that in XboxController because it's much more method calls.